Makefile

```diff
@@ -23,6 +23,19 @@ BUILD := .build
 # usually unnecessary to clean, and may require downloads to restore, so this folder is not automatically cleaned.
 BIN := .bin
 
+# current (when committed) version of Go used in CI, and ideally also our docker images.
+# this generally does not matter, but can impact goimports or formatting output.
+# for maximum stability, make sure you use the same version as CI uses.
+#
+# this can _likely_ remain a major version, as fmt output does not tend to change in minor versions,
+# which will allow findstring to match any minor version.
+EXPECTED_GO_VERSION := go1.16.3
```

Comment:
can probably change this to `go1.16` so it allows all 1.16 minor versions.  unless other 1.16 versions behave differently?